packages/fx-core/resource/yaml-schema.json

```diff
@@ -0,0 +1,1495 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "projectId": {
+      "type": "string",
+      "description": "projectId used for telemetry"
+    },
+    "environmentFolderPath": {
+      "type": "string",
+      "description": "path to the environment folder"
```

Comment:
Some descriptions (like on line 16) start with capitalized words, and others do not. Which is it?

Reply:
All descriptions have been changed to start with capitalized words.